templates/shell.html

```diff
@@ -0,0 +1,208 @@
+{% extends "base.html" %}
+
+{% set loading_overlay_duration = 18000 %}
+{% set host = "https://jupyterlite.github.io" %}
+{% set path = "demo/repl/" %}
```

Comment:
Looks like this can be updated now that the repo has been transferred?

```suggestion
{% set host = "https://sympy.github.io" %}
{% set path = "live/repl/" %}
```

Reply:
Switch to JupyterLite at http://sympy.github.io/live/